apix-impl/build.gradle

```diff
@@ -4,6 +4,31 @@ plugins {
     id 'eu.xenit.alfresco' version '1.0.0' apply false
 }
 
+String getMaxRepositoryVersionRequirement(Project currentProject, String minAlfrescoVersion) {
+    Comparator<Project> comparator = new Comparator<Project>() {
+        @Override
+        int compare(Project p1, Project p2) {
+            return p1.name.compareTo(p2.name)
+        }
+    }
+
+    List<Project> subProjectList = subprojects.toSorted(comparator)
+
+    if (currentProject.getName().equals(subProjectList.last().getName())) {
+        return ""
+    }
+
+    return minAlfrescoVersion + ".99"
+}
+
+ext {
+    alfresco_50_min_version = "5.0"
```

Comment:
The versioning is already spread out over 5 files, so I prefer not to add another one. Can you remove these constants and simply hardcode the versions in the respective overrides.gradle?

Reply:
OK. In that case we can also just remove or set the max_version in overrides.gradle to what it needs to be.
Unless, you want setting the max version to be automated when support for a new Alfresco version is added

Reply:
To be clear, can the getMaxRepositoryVersionRequirement method be kept or should I also hard-code the max Alfresco version?

Reply:
That's also ok for me.

Reply:
To answer Koen: no, no automation is needed for that.

Reply:
@WimCrols OK. So we set (hard-code) min and max Alfresco version in the override.gradle files?

Reply:
Yes